packages/flame_forge2d/example/analysis_options.yaml

```diff
@@ -0,0 +1 @@
+include: package:flame_lint/analysis_options.yaml
```

Comment:
we should have a linter check to enforce this on all packages!

Reply:
Haha, is that even possible? Lintception

Reply:
well via a bash script. not using the flutter linter :P